### DISCUSSION TITLE
Does Chakra have a user-resizeable/adjustable grid system?

### DISCUSSION THREAD

**Discussion by somguyth** [chakra-ui/chakra-ui#4378] [General]
I'm reading https://chakra-ui.com/docs/layout/grid and can't see anything that allows users to resize grid components in the browser.

Is this part of the Chakra featureset? If not, do we know of a library that can achieve this that plays nice with Chakra?

**Comment by anubra266** [comment]
Could you give a use case for better understanding of what you're trying to achieve.

**Reply by somguyth** [reply]
Sure thing.

I have 2 tiled financial charts. 1 row, 2 columns. Some users want to make the 1st chart take up less horizontal room than the 2nd by clicking and dragging.

If you want to see the kind of UI I'm talking about, check out https://www.bitmex.com/app/trade/XBTUSD

I don't want something that complicated, but some ability for the user to change the size of the tiles using the mouse would be nice.

**Reply by anubra266** [reply]
Something like this: https://tomkp.github.io/react-split-pane/?path=/story/basic--percentage-vertical?

**Reply by somguyth** [reply]
> Something like this: https://tomkp.github.io/react-split-pane/?path=/story/basic--percentage-vertical?

Looks like it doesn't work with Chakra-UI ?

I've created a Q&A here: https://github.com/chakra-ui/chakra-ui/discussions/4381
